src/components/renderer/form-masked-input.vue

```diff
@@ -13,6 +14,17 @@
       type="text"
```

Comment:
`formatFloatValue` above can be passed in directly; usually calling the function will place its return value instead of a reference to the actual function. We also usually use the `@` shorthand for event listeners, which I think it's worth sticking to, especially since there's an `@input` on the same component; looks a little odd mixing the two.

```
@blur="formatFloatValue"
```

Reply:
formatFloatValue is not required anymore. It will be removed